app/api/product.py

```
@@ -1,167 +1,262 @@
+            },
+            "capabilities": {
+                "individual_embedding_generation": True,
+                "product_text_preprocessing": True,
+                "category_mapping": True,
+                "price_analysis": True
+            }
+        }
+        
+    except Exception as e:
+        logger.error(f"상품 서비스 상태 조회 실패: {e}")
+        raise HTTPException(status_code=500, detail=f"서비스 상태 조회 실패: {str(e)}")
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)